test/parallel/test-cluster-server-restart-none.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+cluster.schedulingPolicy = cluster.SCHED_NONE;
+
+if (cluster.isMaster) {
+  const worker1 = cluster.fork();
+  worker1.on('listening', () => {
+    const worker2 = cluster.fork();
+    worker2.on('exit', (code) => {
+      assert.equal(code, 0);
+      worker1.kill();
+    });
+  });
+} else {
+  const net = require('net');
+  const server = net.createServer();
+  server.listen(common.PORT, common.mustCall(() => {
+    if (cluster.worker.id === 2) {
+      server.close(() => {
+        server.listen(common.PORT, common.mustCall(() => {
+          server.close();
+          process.exit();
```

Comment:
This `process.exit()` call isn't strictly necessary, is it?  Wouldn't `process.disconnect()` suffice?